taskcluster/scripts/pyenv-setup

```diff
@@ -3,6 +3,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+set -e
+
 curl -L "https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer" -o /builds/worker/pyenv-installer
```

Comment:
Improvement unrelated to this PR: We should log out what revision we pulled from `master` for the sake of traceability.

Reply:
Good suggestion, I added another commit with this + `pyenv --version`.